test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -218,13 +225,21 @@ std::string RandomCode::generate(int _maxOpNumber, RandomCodeOptions _options)
 	for (auto i = 0; i < size; i++)
 	{
 		uint8_t opcode = weightsDefined ? randOpCodeWeight() : randOpCodeGen();
-		dev::eth::InstructionInfo info = dev::eth::instructionInfo((dev::eth::Instruction) opcode);
+		eth::Instruction inst = (eth::Instruction) opcode;
+		eth::InstructionInfo info = eth::instructionInfo(inst);
 
-		if (info.name.find("INVALID_INSTRUCTION") != std::string::npos)
+		if (info.name.find("INVALID_INSTRUCTION") != std::string::npos || info.name.empty()
+			|| std::find(invalidOpcodes.begin(), invalidOpcodes.end(), inst) != invalidOpcodes.end())
 		{
-			//Byte code is yet not implemented
-			if (_options.useUndefinedOpCodes == false)
+			if (_options.useUndefinedOpCodes == true)
+			{
+				std::string byte = toCompactHex(opcode);
+				code += (byte == "") ? "00" : byte;
```

Comment:
IIRC the last parameter for `toCompactHex` already does this - pass 2 there to get always at least 2 symbols